File: db4oj.tests/src/com/db4o/db4ounit/common/ta/mixed/MixedArrayItem.java
package com.db4o.db4ounit.common.ta.mixed;

import com.db4o.db4ounit.common.ta.*;

public class MixedArrayItem {
    
	public Object[] objects;

	public MixedArrayItem() {

	}

	public MixedArrayItem(int v) {
		objects = new Object[4];
		objects[0] = LinkedList.newList(v);
		objects[1] = new TItem(v);
		objects[2] = LinkedList.newList(v);
		objects[3] = new TItem(v);
	}
}

File: db4o.net/Db4objects.Db4o.Tests/Db4objects.Db4o.Tests/Common/TA/Mixed/MixedArrayItem.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using Db4objects.Db4o.Tests.Common.TA;
using Db4objects.Db4o.Tests.Common.TA.Mixed;

namespace Db4objects.Db4o.Tests.Common.TA.Mixed
{
	public class MixedArrayItem
	{
		public object[] objects;

		public MixedArrayItem()
		{
		}

		public MixedArrayItem(int v)
		{
			objects = new object[4];
			objects[0] = LinkedList.NewList(v);
			objects[1] = new TItem(v);
			objects[2] = LinkedList.NewList(v);
			objects[3] = new TItem(v);
		}
	}
}
